Update device_topology_map.md (#23770)
DataDog/documentation: content/en/network_monitoring/devices/device_topology_map.md

### Prerequisites

> 1. Devices have LLDP (Link Layer Discovery Protocol) and/or CDP (Cisco Discovery Protocol) enabled with SNMP. The same protocol should be used on any two connected devices for them to discover each other. LLDP is generally preferred as it is a more common option.

Reviewer: ```suggestion
1. Devices have LLDP (Link Layer Discovery Protocol) and/or CDP (Cisco Discovery Protocol) enabled with SNMP. Use the same protocol on connected devices so that they can discover each other. LLDP is generally preferred as it is a more common option.
```

### Inspect

Choose **Inspect** to see the device's interface connections. Click on any of the connected interfaces for further investigation.

> This view only shows the physical interfaces that are actually connected to another device. In practice this means that it shows a subset of the total set of interfaces of a network device.

Reviewer: ```suggestion
This view shows only the physical interfaces that are actually connected to another device. This means that it shows a subset of the total set of interfaces of a network device.
```
